raml-headers-form.html

```diff
@@ -197,7 +223,7 @@
                     <label>Header name</label>
                     <input
                       is="anypoint-input"
-                      class="name-change-input"
+                      class="name-change-input anypoint-text-input"
```

Comment:
We have text inputs with light gray bg and darker left and right borders, not top and bottoms ones.

<img width="804" alt="screenshot 2017-05-25 16 02 54" src="https://cloud.githubusercontent.com/assets/7399928/26453321/a31b20d4-4163-11e7-8249-b543f9eebb9d.png">